transformer_engine/pytorch/tensor/quantized_tensor.py

```diff
@@ -59,7 +65,7 @@ def update_usage(
             f"{self.__class__.__name__} class does not implement update_usage function"
         )
 
-    def get_usage(self) -> Tuple[bool, bool]:
+    def get_usage(self) -> Dict[str, bool]:
```

Comment:
Nit: A tensor may have multiple usages.
```suggestion
    def get_usages(self) -> Dict[str, bool]:
```